apps/admin-ui/components/Layout/LanguageSwapper.tsx

```diff
@@ -0,0 +1,59 @@
+import { TextField, MenuItem, Box } from "@mui/material";
+import { useLanguage } from "@easy-messe/libs/theme";
+import languageIcon from '@iconify-icons/material-symbols/language'
+import { Icon } from "@iconify/react";
+
+type CorrespondingShortLanguage = Record<string, string>;
+
+export default function LanguageSwapper() {
+    const correspondingShortLanguage: CorrespondingShortLanguage = {
+        'fr': 'Français',
+        'en': 'Anglais'
+    }
```

Comment:
the languages should present in the required format.
```suggestion
    const correspondingShortLanguage: CorrespondingShortLanguage = {
        'fr': 'Français',
        'en': 'English'
    }
```

this permits english to always display like that and read and understood by all.